packages/backend/package.json

```diff
@@ -1,90 +1,86 @@
-    "pretypeorm:cli": "(rimraf  ormconfig.json | echo 'generate:') && ts-node -r tsconfig-paths/register src/config/scripts/write-type-orm-config.ts",
-
-    "db:fill":"npm run typeorm:cli -- migration:run",
-    "db:revertLast":"npm run typeorm:cli -- migration:revert"
-  },
-  "dependencies": {
-    "@nestjs/common": "^6.7.2",
-    "@nestjs/core": "^6.7.2",
-    "@nestjs/microservices": "^6.11.5",
-    "@nestjs/platform-express": "^6.7.2",
-    "@nestjs/swagger": "^4.2.3",
-    "@nestjs/typeorm": "^6.2.0",
-    "@nestjs/websockets": "^6.11.5",
-    "@project/common": "*",
-    "bcrypt": "^4.0.1",
-    "csurf": "^1.11.0",
-    "class-transformer": "^0.2.3",
-    "class-validator": "^0.11.0",
-    "dotenv": "^8.2.0",
-    "fastify-swagger": "^2.5.0",
-    "helmet": "^3.21.3",
-    "passport": "^0.4.1",
-    "passport-jwt": "^4.0.0",
-    "pg": "^7.18.1",
-    "reflect-metadata": "^0.1.13",
-    "rimraf": "^3.0.0",
-    "rxjs": "^6.5.3",
-    "swagger-ui-express": "^4.1.3",
-    "typeorm": "^0.2.22"
-  },
-  "devDependencies": {
-    "@nestjs/cli": "^6.9.0",
-    "@nestjs/schematics": "^6.7.0",
-    "@nestjs/testing": "^6.7.1",
-    "@types/express": "^4.17.1",
-    "@types/jest": "^24.0.18",
-    "@types/node": "^12.7.5",
-    "@types/supertest": "^2.0.8",
-    "@types/passport": "^1.0.2",
-    "@types/passport-jwt": "^3.0.3",
-    "jest": "^24.9.0",
-    "prettier": "^1.18.2",
-    "supertest": "^4.0.2",
-    "ts-jest": "^24.1.0",
-    "ts-loader": "^6.1.1",
-    "ts-node": "^8.4.1",
-    "tsconfig-paths": "^3.9.0",
-    "tslint": "^5.20.0",
-    "typescript": "^3.6.3",
-    "webpack-node-externals": "^1.7.2"
-  },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
-    "rootDir": "src",
-    "testRegex": ".spec.ts$",
-    "transform": {
-      "^.+\\.(t|j)s$": "ts-jest"
-    },
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
-  }
-}
-
+{
+  "name": "@project/backend",
+  "version": "0.0.1",
+  "description": "",
+  "author": "devLeaders",
+  "license": "MIT",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "nest build --watch --webpack",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nodemon --config nodemon.json",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm:cli": "ts-node -r tsconfig-paths/register ./node_modules/typeorm/cli",
+    "pretypeorm:cli": "(rimraf  ormconfig.json | echo 'generate:') && ts-node -r tsconfig-paths/register src/config/scripts/write-type-orm-config.ts",
+    "db:fill": "npm run typeorm:cli -- migration:run",
```

Comment:
czemu masz tu zmiany?

Reply:
przy megrowaniu z masterem to poprawie, bo w sumie to dziwne, że mi tak cały plik zmieniło